src/core/trulens/core/app/base.py

```diff
@@ -676,32 +652,9 @@ def select_context(cls, app: Optional[Any] = None) -> Lens:
         access the retrieved contexts that would appear in a record were these
         components to execute.
         """
```

Comment:
Logic Change: keeping `App.select_context` abstract for subclasses to implement.